GitTfs/Util/ChangeSieve.cs

```diff
@@ -77,7 +77,7 @@ public IEnumerable<IChange> GetChangesToFetch()
             return NamedChanges.Where(c => IncludeInFetch(c)).Select(c => c.Change);
         }
 
-        public IEnumerable<ApplicableChange> GetChangesToApply()
+        public IEnumerable<ApplicableChange> GetChangesToApply(bool force = false)
```

Comment:
I did.

Also I did changes to make sure only first changes set will be forced.

 

 

With Best Regards

 <mailto:zabavnov@gmail.com> Vadim Zabavnov

 

From: Philippe Miossec [mailto:notifications@github.com] 
Sent: Monday, May 18, 2015 12:14 PM
To: git-tfs/git-tfs
Cc: Vadim Zabavnov
Subject: Re: [git-tfs] #772 - fix the issue when first change set is branch (#774)

 

In GitTfs/Util/ChangeSieve.cs <https://github.com/git-tfs/git-tfs/pull/774#discussion_r30522340> :

> @@ -77,7 +77,7 @@ public IEnumerable<IChange> GetChangesToFetch()
>              return NamedChanges.Where(c => IncludeInFetch(c)).Select(c => c.Change);
>          }
>  
> -        public IEnumerable<ApplicableChange> GetChangesToApply()
> +        public IEnumerable<ApplicableChange> GetChangesToApply(bool force = false)

perhaps, use the same name here...

—
Reply to this email directly or view it on GitHub <https://github.com/git-tfs/git-tfs/pull/774/files#r30522340> .  <https://github.com/notifications/beacon/ACjG1ARSdHRKNwVO6PhfJjmur8vgLdYaks5oKgc3gaJpZM4EfBnG.gif>